src/components/map/PreviewFeaturesService.js

```diff
@@ -16,21 +16,21 @@ goog.require('ga_styles_service');
    * Used by Tooltip, FeatureTree, Search and Permalink.
    */
   module.provider('gaPreviewFeatures', function() {
-    var MINIMAL_EXTENT_SIZE = 1965;
-    var highlightedFeature, onClear, listenerKeyRemove, listenerKeyAdd;
-    var geojson = new ol.format.GeoJSON();
-    var source = new ol.source.Vector();
-    var vector = new ol.layer.Vector({
-      source: source
-    });
```

Comment:
Yes these variables were created even if we don't use the gaPreviewFeatures service.